.travis.yml

```diff
@@ -24,26 +18,48 @@ branches:
 matrix:
     fast_finish: true
     include:
+          # Minimum supported Symfony version and latest PHP version
+        - php: 7.1
+          env: DEPENDENCIES="minimum" COVERAGE=true TEST_COMMAND="composer test-ci" SYMFONY_DEPRECATIONS_HELPER="weak" SYMFONY_PHPUNIT_VERSION="5.7"
+
+          # Test the latest stable release
         - php: 5.5
```

Comment:
Not sure you need this job. There are two other jobs making use of PHP 5.5.

Reply:
Hm, maybe you are correct.